frontend/src/SnippetManager.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { ImageProvider } from "./ImageProvider";
+import { SocialMediaAccount } from "./SocialMediaAccountsListTile";
+
+export interface Param {
+    name: string
+    value: string
+}
+
+/**Contains a bunch of useful snippets to be standardized across the application. */
+export class SnippetManager {
+
+    /**Create an info message with a documentation link and a tooltip. */
+    static createInfo(url: string, text: string, cssClasses?: string, tooltip?: string, cssClassesToolTipText?: string) {
+        const className = "tooltip " + cssClasses;
+        const classNameText = "tooltiptext " + cssClassesToolTipText;
+        return <span className={className}>{text}
+            {text && <span><a target="blank" href={url}>here</a>.&nbsp;</span>}
+            <a target="blank" href={url}>{ImageProvider.getImage("info-text")}</a>
+            <span className={classNameText}>{tooltip ? tooltip : text}</span>
+        </span>
+    }
+
+    /**Create an info message with a documentation link without a tooltip. */
+    static createInfoWithoutTooltip(url: string, text: string, cssClasses?: string) {
+        const className = "" + cssClasses;
+        return <span className={className}>{text}
+            {text && <span><a target="blank" href={url}>here</a>.&nbsp;</span>}
```

Comment:
Did you wrap it inside a span to have a single child? This could be solved better using [`React.Fragment`](https://reactjs.org/docs/react-api.html#reactfragment) (=&nbsp;[`DocumentFragment`](https://developer.mozilla.org/en-US/docs/Web/API/DocumentFragment)):
```suggestion
            {text && <><a target="blank" href={url}>here</a>.&nbsp;</>}
```

Reply:
Yes.